docs/migrations/guides/v1.1.0-to-v1.2.0.md

```
@@ -0,0 +1,136 @@
+    collections: [
+      'lucide',
+      'mdi',
+      'simple-icons',
+      'noto',
+    ],
+  },
+})
+```
+
+Rename legacy `i-noto-*` values with these commands:<br>
+([Step 1 (icon name renaming)](#1-migrate-icon-names-to-prefixname) for `noto` icons.)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)